src/styles/style.css

```diff
@@ -3,6 +3,8 @@
 body {
   width: 100vw;
   height: 100vh;
```

Comment:
- [ OPTIONAL] Please avoid unnecessary code that doesn't affect the project because it overloads your project.